examples/plotting/file/line_toggle.py

```diff
@@ -0,0 +1,40 @@
+"""
+Example demonstrating the picking of line objects. The transparency of
+the lines is toggled when clicked.
+"""
+
+import numpy as np
+from bokeh.plotting import output_file, show, figure, Figure
+from bokeh.models import TapTool, CustomJS
+
+# Normally, the use of the tap tool causes Bokehjs to "select" a line
+# when it is clicked on, which is not what we want in this example. To
+# deal with this, we modify the "selected" object to make it look as
+# if no glyph was clicked.
+
+code = """
+d0 = cb_obj.get("selected")["0d"];
+if (d0.glyph) {
+    var alpha = d0.glyph.visuals.line.alpha.value();
+    var color = d0.glyph.visuals.line.color.value();
+    console.log('Toggled line', d0.glyph.id, color);
+    d0.glyph.visuals.line.alpha.fixed_value = (alpha == 1) ? 0.5 : 1.0;
+    // Make selected/unselected lines have normal appearance
+    d0.glyph['renderer'].selection_glyph = null;
+    d0.glyph['renderer'].nonselection_glyph = null;
```

Comment:
I'm not sure I follow, I would be surprised if setting `d0.glyph = null` works in any way, but even if it does something besides fall over, it still seems like an unusual and hacky workaround, not something I'd want to codify and demonstrate. I'd like to merge this PR without an example, and make the change I suggested above for `"auto"` in another PR.